recipes/spdlog/conandata.yml

```diff
@@ -0,0 +1,40 @@
+sources:
+  "1.10.0":
+    url: "https://github.com/gabime/spdlog/archive/v1.10.0.tar.gz"
+    sha256: "697f91700237dbae2326b90469be32b876b2b44888302afbc7aceb68bcfe8224"
```

Comment:
Should I check the hash values?

Reply:
This is copied from https://github.com/conan-io/conan-center-index/blob/master/recipes/spdlog/all/conandata.yml